README.md

```diff
@@ -1,24 +1,26 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
 
-Things you may want to cover:
+# honestreviews.com
 
-* Ruby version
+honestreviews is an website for users to find honest up to date information on schools they might be interested in attending. We use ruby on rails and postgres for the backend and use React for the website front end. 
```

Comment:
A few small recommended changes:
```suggestion
Honestreviews is a website for users to find honest, up-to-date information on schools they might be interested in attending. 
```
Keep in mind that while the front end is React, the application build is React in Rails.  The way this is worded appears more like this is a decoupled app which it is not.